api/templates/deploy.sh.hbs

```diff
@@ -1,29 +1,33 @@
 #!/usr/bin/env bash
-set -e
+cd "$(dirname "${BASH_SOURCE[0]}")"
+set -eu
 
-stack_name="$1"
-common_stack_name="${2:-{{ kebabCase criName }}-common-cri-api-local}"
-secret_prefix="${3:-{{ kebabCase criName }}-cri-api}"
+stack_name="${1:-}"
+common_stack_name="${2:-}"
 
-if [ -z "$stack_name" ]
-then
-echo "😱 stack name expected as first argument, e.g. ./deploy {{ kebabCase criName }}-user1"
-exit 1
+if ! [[ "$stack_name" ]]; then
+  [[ $(aws whoami --query Arn --output text) =~ \/([^\/\.]+)\. ]] && user="${BASH_REMATCH[1]}" || exit
+  stack_name="$user-{{ kebabCase criNameShort }}"
+  echo "» Using stack name '$stack_name'"
 fi
```

Comment:
When I tried this, it was deploying a stack with my name when left out, i get the below

```
AWS_PROFILE=cri-check-hmrc-dev npm run deploy         

> deploy
> ./deploy.sh


usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
```

Reply:
@jkunle do you have an alias file in your AWS directory? There shoule be a file called `~/.aws/cli/alias` with the entry `whoami = sts get-caller-identity`.

I don't remember if it's created by default but if it's not I'll change the script to use the full command instead.